crypto/dilithium/p_pqdsa_test.cc

```diff
@@ -1343,42 +1434,97 @@ TEST_P(PQDSAParameterTest, SIGOperations) {
       0x4a, 0x41, 0x4b, 0x45, 0x20, 0x4d, 0x41, 0x53, 0x53, 0x49,
       0x4d, 0x4f, 0x20, 0x41, 0x57, 0x53, 0x32, 0x30, 0x32, 0x31, 0x2e};
 
-  ASSERT_TRUE(EVP_DigestSignInit(md_ctx.get(), nullptr, nullptr,
-                                 nullptr, pkey.get()));
-  ASSERT_TRUE(EVP_DigestSign(md_ctx.get(), signature1.data(), &sig_len,
-                             msg1.data(), msg1.size()));
+  // ---- 2. Test signature round trip (sign + verify) ----
+
+  // Initalize the signing context |md_ctx| with the |pkey| we generated
```

Comment:
If you want, you can replace in the comments
message1 -> msg1
message2 -> msg2

Reply:
fixed in [f7d1dad](https://github.com/aws/aws-lc/pull/2062/commits/f7d1dadb64f54b7695d871a88d4dfacf42048637)